smoke_test/mail_box.rb

```diff
@@ -0,0 +1,35 @@
+require 'mail'
+require 'net/imap'
+
+module SmokeTest
+  module MailBox
+    SMOKE_TEST_EMAIL_PASSWORD = ENV.fetch 'SMOKE_TEST_EMAIL_PASSWORD'
+    SMOKE_TEST_EMAIL_HOST = ENV.fetch 'SMOKE_TEST_EMAIL_HOST'
```

Comment:
Oops, so they are. The fact that they deceive the eye with their similarity is possibly a warning flag.